cmd/import.go

```diff
@@ -56,25 +55,24 @@ type exportJobCreateRequest struct {
 	Tables      []string `json:"tables,omitempty"`
 }
 
-func shouldRetryError(err error) bool {
-	msg := err.Error()
-	if strings.Contains(msg, "connection refused") || strings.Contains(msg, "connection reset") {
-		return true
-	}
-	return false
+type errorResponse struct {
+	Message string `json:"message"`
 }
 
-func shouldRetryStatus(statusCode int) bool {
-	switch statusCode {
-	case http.StatusInternalServerError, http.StatusServiceUnavailable, http.StatusGatewayTimeout, http.StatusTooManyRequests:
-		return true
-	default:
-		return false
+func (e *errorResponse) Parse(buf []byte, statusCode int) error {
+	if err := json.Unmarshal(buf, e); err == nil {
+		return fmt.Errorf(e.Message)
 	}
+	return fmt.Errorf("API error: %s (status code=%d)", string(buf), statusCode)
 }
 
-func backoffRetry(retryCount int) time.Duration {
-	return time.Duration(rand.Intn(1000*retryCount)) * time.Millisecond
+func handleAPIError(resp *http.Response) error {
```

Comment:
@robindiddams once your PR is in i will refactor all of API results to handle error message with this...